lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -55,6 +59,7 @@ def compute(self, student, course_structure, scores_client, submissions_scores):
             # self.scores may hold outdated data, force it to refresh on next access
             lazy.invalidate(self, 'scores')
 
+        log.info("accessed lazy property scores with value: {0}".format(unicode(self.scores)))
```

Comment:
I'm not sure that it's worth logging each access to these properties - even though they are `lazy`.  Rather than logging these, we should have unit tests to capture proper handling/invalidating of these `lazy` attributes.